clang/docs/ReleaseNotes.rst

```diff
@@ -392,6 +392,8 @@ related warnings within the method body.
 - Clang now disallows the use of attributes applied before an
   ``extern template`` declaration (#GH79893).
 
+- Clang now diagnoses unknown attribute namespaces.
```

Comment:
I would move this to improvements to clang's diagnostics, and reword it to: Clang now prints the namespace for an attribute, if any, when emitting an unknown attribute diagnostic.